scripts/konduit.sh

```diff
@@ -123,6 +124,64 @@ init_setup() {
 
    # Set service ports
    DB_PORT=5432
+
+   # Set variables if using a separate deployment to access the database
+   if [ "${Jumppod}" != "" ]; then
+      OLDINST=${INSTANCE}
+      INSTANCE="konduit-app-${RANDOM}"
+      cat - << EOF | sed s/APPNAME/${INSTANCE}/ > jumppod.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: APPNAME
+  labels:
+    app: konduit-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: konduit-app
+  template:
+    metadata:
+      labels:
+        app: konduit-app
+    spec:
+      automountServiceAccountToken: false
+      nodeSelector:
+        teacherservices.cloud/node_pool: applications
+        kubernetes.io/os: linux
+      containers:
+      - name: konduit-container
+        image: alpine:3.20.1
+        command: ['sh', '-c', 'sleep 10800']
+        securityContext:
+          runAsUser: 1000
+          runAsGroup: 3000
+          capabilities:
+            add:
+              - NET_BIND_SERVICE
+            drop:
+              - ALL
+          allowPrivilegeEscalation: false
+          privileged: false
+          runAsNonRoot: true
+          readOnlyRootFilesystem: true
+          seccompProfile:
+            type: RuntimeDefault
+        resources:
+          requests:
+            cpu: 10m
+            memory: 50Mi
+          limits:
+            cpu: 100m
+            memory: 50Mi
+        ports:
+        - containerPort: 80
+EOF
+      echo "Using app ${INSTANCE} to connect to database for ${OLDINST}"
+      kubectl -n ${NAMESPACE} create -f jumppod.yaml
```

Comment:
We could avoid creating the jumppod.yaml file so we don't have to delete later on. Store the yaml in a variable and pass it via stdin like:
```
echo $VAR | kubectl -n ${NAMESPACE} create -f -
echo $VAR | kubectl -n ${NAMESPACE} delete -f -
```

Reply:
Had to change to json, but now works as a var